Source/BugsnagClient.m

```diff
@@ -196,7 +198,9 @@ void BSSerializeDataCrashHandler(const BSG_KSCrashReportWriter *writer, int type
  *
  * @returns A string representing the device orientation or nil if there's no equivalent
  */
-#if TARGET_OS_IOS
+
+#if BSG_PLATFORM_TVOS
```

Comment:
Why the empty test for tvOS? If it's to show that iOS only is intentional, we should probably also include an empty macOS block.

Reply:
This was required when `BSG_PLATFORM_IOS` was defined via the simulator macro - this is no longer required so has been amended.